### ISSUE TITLE
[2.0.0] Chrony Restart Error

### ISSUE THREAD

**Issue by kylegoch** [geerlingguy/ansible-role-ntp#73]
We let our version of this module float, however with the recent bump to version 2.0 we are seeing an error at the end of our Ansible run relating to the module.

We are running on Centos 7.7 and get this error:
```
     amazon-ebs-kubernetes: RUNNING HANDLER [geerlingguy.ntp : restart ntp] ********************************
     amazon-ebs-kubernetes:
     amazon-ebs-kubernetes: fatal: [default]: FAILED! => {"changed": false, "msg": "Could not find the requested service chronyd: host"}
```
(log is from Packer, thus formatting)

We made no changes to our variables with regards to NTP. Rolling back to 1.6.4 fixed the issue.
I saw #72 but it seems that should have just worked. As we dont care whether we use chrony or ntp.

**Comment by JuddTracy-DAS** [comment]
I am having the same issue.  To me it looks like the problem is that the var ntp_daemon is defined as chronyd in the vars/Redhat.yml and because he did not do the __<var_name> set_fact mapping that he seems to be switching to the variable cannot be redefined in group_vars or host_vars.

**Comment by JuddTracy-DAS** [comment]
@kylegoch I submitted a pull request last night that hopefully will fix the issue you have.

I also did some more digging into the issues since I had some systems that reported it was working and other systems where it broke.  Turns out that even if you specify the config file for ntp the role will always try to setup chrony in systemd (on RHEL at least).  The reason why it broke on some of my systems but not others is that chrony was removed on some after install so when the broken logic tries to start chrony even though I wanted ntp it would fail because the daemon does not exist.
